kit/feature/list.go

```diff
@@ -58,16 +58,32 @@ func SessionService() BoolFlag {
 	return sessionService
 }
 
+var redirectto = MakeBoolFlag(
```

Comment:
is this autogenerated code from `make flags` that you forgot to remove?

Reply:
it is autogenerated code that I didn't realize was autogenerated ;)